### DISCUSSION TITLE
Need more example

### DISCUSSION THREAD

**Discussion by linnaname** [vectordotdev/vector#9899] [General]
I used vector a few weeks ago, it's great. 

But I think it may need more examples. These  examples is more difficult  than the document example we use now. because  sometimes  I don't know how to combine component when the logic is complex. And may need some best practice example.

**Comment by jszwedko** [comment]
Hi @linnaname ! Thanks for the feedback. We are aware that we need more examples, in addition to the reference documentation. This is something we hope to improve on over the next 6-12 months. We are glad to hear that others would find this useful!

**Comment by pront** [comment]
Ref https://github.com/vectordotdev/vector/issues/21735
